shared/src/components/Common/TableWithFixedHeader.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useMemo, useRef } from "react";
+import styled from "styled-components";
+import { ExternalIcon } from "../../assets/icons/icons";
+import { BaseLink, SecondaryText } from "../../designSystem";
+import colors from "../../designSystem/colors";
+import sizes from "../../designSystem/sizes";
+import theme from "../../designSystem/theme";
+
+import useElementSize from "../../hooks/useElementSize";
+
+const logoContainerWidth = 40;
+const logoContainerMargin = 16;
+const linkContainerWidth = 20;
+const linkContainerMargin = 28;
+
+const TableHeader = styled.div`
+  display: flex;
+  margin-top: 24px;
+  padding: 0px 16px;
```

Comment:
The margin-bottom can be decreased here
<img width="1191" alt="Screenshot 2021-07-13 at 4 49 16 PM" src="https://user-images.githubusercontent.com/5748750/125483932-f2c110f6-fb64-426c-8d78-3f2ed7142233.png">